plugin-help/che-plugin-help-ext-client/src/main/resources/org/eclipse/che/ide/ext/help/client/About.css

```diff
@@ -0,0 +1,42 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2015 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+@eval logoColor org.eclipse.che.ide.api.theme.Style.getLogoColor();
+
+.emptyBorder {
+    margin: 6px;
+}
+
+.label {
+    margin: 6px 5px 0 15px;
+    margin-right: 5px;
+    font-size: 11px;
+    font-weight: bold;
+    float: left;
```

Comment:
Use DockLayoutPanel in ui binder template instead of floating labels to left. You'll have less problems in future with supporting your code